specs/2024-10-30.yaml

```diff
@@ -2418,7 +2418,6 @@ components:
       required:
         - organization_id
         - amount
-        - payment_id
```

Comment:
remove payment_id as a requirement. This payment id is only relevant if the org creates the payment through the CreatePayments API, but if they choose to process the payments themselves, there is no payment ID to provide